### PR TITLE
zh:sync announcing-1.15.2

### PR DESCRIPTION
**Pull request by liangyongzhenya**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+标题。宣布Istio 1.15.2发布
```
Comment (rootsongjc) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:2]: 这些 Hugo 文档的配置字段不需要翻译，否则文档将无法渲染。

**Review by zirain — CHANGES_REQUESTED** [review]
如果不熟悉工作流，请先参考别人的做法。例如：
https://github.com/istio/istio.io/pull/12061

最好能够提供一个友好的标题

**Review by windsonsea — COMMENTED** [review]
请先修改几个明显的问题：
- 中英之间空格
- 式样与 en 需一致

参阅 https://github.com/istio/istio.io/pull/12028/files

**Comment by windsonsea** [comment]
Thanks, you may need to perform the following steps before review:
1. Sign the cla/google: https://github.com/istio/istio.io/pull/12064/checks?check_run_id=8784849567
2. Fix the lint errors: https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12064/lint_istio.io/1578944780321165312

**Comment by ericvn** [comment]
Setting DNM until 1.15.2 is released.

**Comment by ericvn** [comment]
Also, the original PR is changing slightly. Some changes need to be reflected here like like 12 which is changes in 1.15.2 from 1.15.1.

**Comment by ericvn** [comment]
1.15.2 RN is now merged in the release branch and soon to be merged in the main branch.

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Istio 1.15.2
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:2]:
```suggestion
title: Istio 1.15.2 发布
```

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+linktitle: 1.15.2
+subtitle: Patch Release
+description: Istio 1.15.2 补丁发布
+publishdate: 2022-10-11
+release: 1.15.2
+---
+
+该版本包括 Go 1.19.2（2022-10-04发布）中对 `archive/tar`、`net/http/httputil` 和 `regexp` 包的安全修复。
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:10]:
```suggestion
该版本包括 Go 1.19.2（2022-10-04 发布）中对 `archive/tar`、`net/http/httputil` 和 `regexp` 包的安全修复。
```

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+这个版本包含了改进鲁棒性的错误修正。
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:11]:
```suggestion
这个版本包含了改进稳健性的漏洞修复。
```

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+本发布说明描述了 Istio 1.15.2 和 Istio 1.15.2 之间的不同之处。
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:12]:
```suggestion
本发布说明描述了 Istio 1.15.1 和 Istio 1.15.2 之间的不同之处。
```

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+{{< relnote >}}}
+
+## 变更{#changes}
+
+-  **修复** 修复了 在1.14.0中，直通集群的默认 "idleTimeout "被改为 "0s"，禁用了超时的问题。恢复了之前的行为，使用 Envoy 的默认值1小时。([问题#41114](https://github.com/istio/istio/issues/41114))
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:18]:
```suggestion
-  **修复** 修复了 在 1.14.0 中 Passthrough 集群的默认 `idleTimeout` 更改为 `0s` 会禁用超时的问题。将其恢复为之前的行为，即使用 Envoy 的默认值 1 小时。([Issue #41114](https://github.com/istio/istio/issues/41114))
```

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+-  **修复** 修复了 网关 API 集成，使其在删除 `v1alpha2` 版本时不会失败。
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:20]:
```suggestion
-  **修复** 修复了移除 `v1alpha2` 版本时网关 API 集成不会失败的问题。
```

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+-  **修复** 修复了 对已废弃的自动缩放设置的处理。([问题 #41011](https://github.com/istio/istio/issues/41011))
```
Comment (windsonsea) [inline content/zh/news/releases/1.15.x/announcing-1.15.2/index.md:22]:
```suggestion
-  **修复** 修复了处理已弃用的自动缩放设置的问题。([Issue #41011](https://github.com/istio/istio/issues/41011))
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
Please try to look details when you handle text job.

github 是职场的延伸，请认真对待每一个字、每一句话

**Comment by windsonsea** [comment]
Please carefully read those comments rather than only pick some to apply. One tip is you can copy/paste the suggested text.

The lint error for your info:
```
ERROR 2022/10/12 13:37:15 Page description doesn't end with a period: 'Istio 1.15.2 补丁发布'
```

**Comment by windsonsea** [comment]
Please check those comments again and click `Resolve conversation` if you have corrected all, or you can give some feedbacks.

File: content/zh/news/releases/1.15.x/announcing-1.15.2/index.md (hunk 8 above)
Comment (windsonsea) [reply content/zh/news/releases/1.15.x/announcing-1.15.2/index.md]: 确定看懂这条 comments 了？会复制粘贴吗？
Issue 不用翻译！

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request closed without merging**